sycl/include/CL/sycl/intel/sub_group.hpp

```diff
@@ -266,7 +266,7 @@ struct sub_group {
   __SYCL_DEPRECATED("Two-input sub-group shuffles are deprecated.")
   T shuffle_down(T current, T next, uint32_t delta) const {
 #ifdef __SYCL_DEVICE_ONLY__
-    return sycl::detail::sub_group::shuffle_down(current, next, delta);
```

Comment:
Do we need to remove definition and declaration of `sub_group::shuffle_down` and other functions which are not used here anymore?

Reply:
Ah, good catch.  Yes, I think we should.

Reply:
Fixed in 221ece8.